### PR TITLE
core: dt_driver: dt_driver: fix nodes reference passed in dt_pargs

### PR DESCRIPTION
**Pull request by etienne-lms**

### CODE REVIEW

**Comment by etienne-lms** [comment]
No strong need to consider this change in tomorrow 4.3.0 release :)

File: core/include/kernel/dt_driver.h [hunk 1]
```diff
@@ -89,12 +89,14 @@ struct dt_driver_provider;
  * struct dt_pargs - Devicetree phandle arguments
  * @fdt: Device-tree to work on
  * @phandle_node: Node pointed by the specifier phandle
+ * @consumer_node: Node of the consumer device requesting this device
```
Comment (GseoC) [inline core/include/kernel/dt_driver.h:92]: "consumer device" may be replaced by "consumer" IMO

**Comment by GseoC** [comment]
1 nit comment, otherwise it seems LGTM, `Acked-by: Gatien Chevallier <gatien.chevallier@foss.st.com>`

**Comment by etienne-lms** [comment]
Inline comment fixed as proposed.
Review tag applied.

**Pull request merged**